src/subscription_manager/managercli.py

```diff
@@ -2660,6 +2662,43 @@ def _list(self, all_overrides, specific_repos):
             print(columnize(names, echo_columnize_callback, *values, indent=2) + "\n")
 
 
+class RoleCommand(CliCommand):
```

Comment:
Forgot to add this to the list of commands [here](https://github.com/candlepin/subscription-manager/blob/master/src/subscription_manager/managercli.py#L2746-L2752).

Reply:
It was there, but it was lost during merging.